### PR TITLE
169645673-accommodation-likes

### PR DESCRIPTION
**Pull request by alexismajya**
#### What does this PR do?

- As a user, I can see the likes of an accommodation facility

#### Description of Task to be completed?
 - Once a user logged into the barefoot nomad, he/she can see the number of accommodation likes

#### How should this be manually tested?

- test this `get` route: `api/v1/accommodations/accommodationId/like`

#### What are the relevant pivotal tracker stories?

[Pivotal story](https://www.pivotaltracker.com/story/show/169645673)

#### Screenshots (if appropriate)

Screen Shot 2019-11-12 at 11 16 19

#### Questions:

### CODE REVIEW

**Review by urbanishimwe — APPROVED** [review]

File: src/routes/accommodation.route.js [hunk 1]
```diff
@@ -29,7 +29,14 @@ app.get('/:accommodationId/rating', valid.validateAccommodationId, BookingsContr
...
 app.get('/:accommodationId', AccommodationsController.getAccommodationById);
-app.post('/:accommodationId/like', checkToken, LikingsController.addLikeAccommdation);
+app.post('/:accommodationId/like', [
+  checkToken,
+  LikingsController.addLikeAccommdation
+]);
+app.get('/:accommodationId/like', [
+  checkToken,
+  LikingsController.getAccommdationLikes
+]);
```
Comment (elchroy) [inline src/routes/accommodation.route.js]: For consistency, avoid using the array format for middlewares.

File: src/tests/liking.accommodation.spec.js [hunk 2]
```diff
@@ -67,4 +67,22 @@ describe('Like/Unlike an accomodation facility', () => {
         done();
       });
   });
+  it('it should return 200  and count the number of likes of the accommodation', (done) => {
+    chai.request(app)
+      .get('/api/v1/accommodations/2/like')
+      .set('Authorization', helper.createToken(2, 'requester@gmail.com', true, 'requester'))
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
```
Comment (elchroy) [inline src/tests/liking.accommodation.spec.js:75]: You need to also expect that the number of likes is correct

File: src/routes/accommodation.route.js (hunk 1 above)
Comment (alexismajya) [reply src/routes/accommodation.route.js]: ok. let me fix them

File: src/tests/liking.accommodation.spec.js (hunk 2 above)
Comment (alexismajya) [reply src/tests/liking.accommodation.spec.js:75]: great

**Review by lemoissonM — APPROVED** [review]

**Pull request merged**